EPRN.Portal/Controllers/PrnsController.cs

```diff
@@ -34,5 +34,22 @@ public async Task<IActionResult> PrnSavedAsDraftConfirmation(int? id)
 
             return View(viewModel);
         }
+
+        [HttpGet]
+        public async Task<IActionResult> Action(int? id)
+        {
+            if (id == null)
+            {
+                return NotFound();
+            }
+            else if (id <= 0)
```

Comment:
Son't need the else (Also since it's a one line content for the if, then the braces are not needed either, and make the whole thing look neater

Reply:
Done.